packages/analyzer/src/features/analyse-phase/creators/handlers.js

```diff
@@ -139,9 +139,15 @@ export function createClassDeclarationMixin(name, moduleDoc) {
 
 /**
  * Handles mixins and superclass
+ + * @param {import('custom-elements-manifest/schema').ClassDeclaration} classTemplate
+ + * @param {import('custom-elements-manifest/schema').Module} moduleDoc
+ + * @param {import('typescript').ClassDeclaration} node
  */
 export function handleHeritage(classTemplate, moduleDoc, node) {
   node?.heritageClauses?.forEach((clause) => {
+    if (clause.token !== ts.SyntaxKind.ExtendsKeyword)
```

Comment:
Can we add a comment above this if statement that we're doing this to ignore `implements` for the time being?

Reply:
done